fvtest/threadtest/createTest.cpp

```diff
@@ -492,12 +492,12 @@ getOsStacksize(size_t stacksize)
 #if defined(LINUX) || defined(OSX)
 	/* Linux allocates 2MB if you ask for a stack smaller than STACK_MIN */
 	{
-		size_t pageSafeMinimumStack = 2 * sysconf(_SC_PAGESIZE);
+		long pageSafeMinimumStack = 2 * sysconf(_SC_PAGESIZE);
 
 		if (pageSafeMinimumStack < PTHREAD_STACK_MIN) {
 			pageSafeMinimumStack = PTHREAD_STACK_MIN;
 		}
-		if (stacksize < pageSafeMinimumStack) {
+		if (stacksize < (size_t)pageSafeMinimumStack) {
```

Comment:
casting an `int64_t` to `size_t` will lead to loss of data. better to use the same data type for `stacksize` and `pageSafeMinimumStack`. 
```suggestion
		if (stacksize < pageSafeMinimumStack) {
```